crates/ruma-common/src/serde/raw.rs

```diff
@@ -11,9 +11,10 @@ use serde::{
 };
 use serde_json::value::{to_raw_value as to_raw_json_value, RawValue as RawJsonValue};
 
-/// A wrapper around `Box<RawValue>`, to be used in place of any type in the Matrix endpoint
-/// definition to allow request and response types to contain that said type represented by
-/// the generic argument `Ev`.
+/// A wrapper around `Box<RawValue>`.
+///
+/// To be used in place of any type in the Matrix endpoint definition to allow request and response
+/// types to contain that said type represented by the generic argument `Ev`.
```

Comment:
Yeah, this sentence is a bit weird. Maybe remove it and add "with a generic parameter for the expected Rust form" or something like that to the first line?

Reply:
Done